### ISSUE TITLE
Remove watched videos from home

### ISSUE THREAD

**Issue by lu4p** [futo-org/grayjay-android#1498]
### Use Cases

I want to discover new content in my home tab.

### Proposal

Videos which have already been watched should be hidden from the home tab to make it easier to find new content.

**Comment by TurtleMeds** [comment]
I agree, and I hate having to go through my home tab clicking hide on every video I've watched, instead of just having a filter like in the subscriptions tab.

**Comment by VoxelPrismatic** [comment]
Close as complete?

<img width="1001" height="1012" alt="Image" src="https://github.com/user-attachments/assets/87624d4f-1eba-4049-9780-880b36943438" />

**Comment by Zvonimir-FUTO** [comment]
@VoxelPrismatic Thank you for helping us with finding these that are done and not closed yet!

**Comment by VoxelPrismatic** [comment]
No problem!
I just sent a job application to FUTO, so hopefully I can help with the UI/UX backlog soon.

*I deleted the previous comment because github mobile rearranged the comments and I thought I was in the wrong ticket

**Comment by koen-futo** [comment]
@VoxelPrismatic Join our public Zulip https://chat.futo.org/#narrow/channel/46-Grayjay and send Zvonomir a message.
